File: financialcalc.c
#include <stdio.h>

int main(void){

    float income, rent, utilities, grocereries, transportation, savings, expenses, spend;

    float prent, putilities, pgroceries, ptransportation, psavings, pexpenses;

    printf("This is a budget calculator. Kind of. What is your monthly income? \n");
    
    scanf("%f", &income);

    printf("How much do your utilities cost per month?");

    scanf("%f", &utilities);

    printf("How much does your rent cost per month?");

    scanf("%f", &rent);

    printf("How much do your groceries cost per month?");

    scanf("%f", &grocereries);

    printf("How much does your transportation cost per month?");

    scanf("%f", &transportation);

    savings = income*0.2;

    expenses = rent+utilities+grocereries+transportation;

    spend = income-expenses-savings;

    printf("You make $%.2f\n", income);

    printf("Your expenses are $%.2f\n", expenses);

    printf("Your savings are $%.2f\n",savings );

    printf("Your spending money is $%.2f\n", spend);

    prent = rent/income*100;

    putilities = utilities/income*100;
    
    pgroceries = grocereries/income*100;

    ptransportation = transportation/income*100;

    psavings = savings/income*100;

    pexpenses = expenses/income*100;

    printf("Your rent is %d%% percent of your total income.\n", (int) prent);

    printf("Your utilities are %d%% percent of your total income.\n", (int) putilities);

    printf("Your groceries are %d%% percent of your total income.\n", (int) pgroceries);

    printf("Your transportation is %d%% percent of your total income.\n", (int) ptransportation);

    printf("Your savings are %d%% percent of your total income.\n", (int) psavings);

    printf("Your expenses are %d%% percent of your total income.\n", (int) pexpenses);
}

File: financialcalc.py
def question(type):
    return int(input(f"Monthly {type}"))

monthlyIncome = question("income")

rent = question("rent")

utilities = question("utilities")

groceries = question("groceries")

transportation = question("transportation")

rentPercent = (rent/monthlyIncome)

utilPercent = (utilities/monthlyIncome)

grocPercent = (groceries/monthlyIncome)

commutePercent = (transportation/monthlyIncome)

savings = (monthlyIncome*0.15)

accountAfter = int((monthlyIncome-(rent+utilities+groceries+transportation+savings)))

expenses = int(rent+utilities+transportation+groceries+savings)

def percent(type, amount):
    per = amount/monthlyIncome*100
    print(f"Your {type} is {per}% income.")

print(f"You should put " + str(savings) + " dollars into your savings account. This is 15 percent of your monthly income.")

print(f"You have " + str(accountAfter) + " dollars left from your monthly income. Have fun!")

print(f"Your monthly income is $" + str(monthlyIncome) + ".")

print(f"Your expenses are $" + str(expenses) + ".")

percent("rent", rent)

percent("groceries", groceries)

percent("utilities", utilities)

percent("commute", transportation)



